### PR TITLE
Issue/89/create buttons styles

### PR DESCRIPTION
**Pull request by Bedpaw**

### CODE REVIEW

File: src/style/components/atoms/button/primary.sass [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  margin: 28px 0 0
+  align-self: center
+
+
+  &:focus
+    opacity: 0.76
+
+  &:hover
+    background: #006027
```
Comment (Bedpaw) [inline src/style/components/atoms/button/primary.sass]: change to $Dark_green

File: src/components/atoms/button/submit.vue [hunk 2]
```diff
@@ -2,6 +2,7 @@
   <a-button
     :disabled="disabled"
     @click="$emit('click')"
+    class="f-primary"
   >
```
Comment (Drego31) [inline src/components/atoms/button/submit.vue]: You have created `a-button-primary` and didn't use it here...
Use it here.

File: src/components/organisms/form/change-password.vue [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="f-pb-1">
-    <div v-if="!formSend">
+    <div v-if="!formSend" class ="f-flex f-flex-col" >
```
Comment (Drego31) [inline src/components/organisms/form/change-password.vue]:
```
<div class="f-pb-1">
  <div v-if="!formSend" class ="f-flex f-flex-col" >
```
These lines are in few files. Create `o-form` component and put the lines there.

File: src/style/components/atoms/button/primary.sass [hunk 4]
```diff
@@ -0,0 +1,29 @@
+.a-button.f-primary
+  @extend .f-text-14, .f-text-center
+  width: 184px
+  height: 36px
+  background: #30B25B
```
Comment (Drego31) [inline src/style/components/atoms/button/primary.sass]: Use color from palete in sass variables.

File: src/style/components/atoms/button/secondary.sass [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  text-transform: uppercase
+  color: $Green
+
+  margin: 28px 0 0
+  align-self: center
+
+
+
+
+
```
Comment (Drego31) [inline src/style/components/atoms/button/secondary.sass]: Too many lines are here

File: src/style/features/display.sass [hunk 6]
```diff
@@ -7,4 +7,3 @@
 
 .f-flex
   display: flex !important
-
```
Comment (Drego31) [inline src/style/features/display.sass]: Each file should has last empty line.
Do not remove that line.

File: src/components/atoms/button/primary.vue [hunk 7]
```diff
@@ -0,0 +1,11 @@
+<template>
+  <button @click="$emit('click')" class="a-button f-primary">
+    <slot/>
+  </button>
```
Comment (Drego31) [inline src/components/atoms/button/primary.vue]: Use `a-button` component here

File: src/components/atoms/button/secondary.vue [hunk 8]
```diff
@@ -0,0 +1,11 @@
+<template>
+  <button @click="$emit('click')" class="a-button f-secondary">
+    <slot/>
+  </button>
```
Comment (Drego31) [inline src/components/atoms/button/secondary.vue]: Use `a-button` component here

File: src/components/atoms/button/primary.vue (hunk 7 above)
Comment (Drego31) [reply src/components/atoms/button/primary.vue]: I've fixed it in last commit

File: src/components/atoms/button/secondary.vue (hunk 8 above)
Comment (Drego31) [reply src/components/atoms/button/secondary.vue]: I've fixed it in last commit

File: src/components/atoms/button/submit.vue (hunk 2 above)
Comment (Bedpaw) [reply src/components/atoms/button/submit.vue]: changed

File: src/style/components/atoms/button/primary.sass (hunk 1 above)
Comment (Bedpaw) [reply src/style/components/atoms/button/primary.sass]: replaced

File: src/style/components/atoms/button/primary.sass (hunk 4 above)
Comment (Bedpaw) [reply src/style/components/atoms/button/primary.sass]: done

File: src/style/components/atoms/button/secondary.sass (hunk 5 above)
Comment (Bedpaw) [reply src/style/components/atoms/button/secondary.sass]: changed to 2 lines

File: src/style/features/display.sass (hunk 6 above)
Comment (Bedpaw) [reply src/style/features/display.sass]: done

File: src/components/organisms/form/change-password.vue (hunk 3 above)
Comment (Drego31) [reply src/components/organisms/form/change-password.vue]: We will create that later

**Comment by Drego31** [comment]
#89 Merged

**Pull request merged**